web/themes/custom/pineapple/components/_card.twig

```diff
@@ -25,9 +25,10 @@
       </div>
     {% endif %}
     {% include "@components/_heading.twig" with {
-      heading_level: 2,
+      heading_level: heading_level,
       heading_url: true,
       heading_text: title,
+      heading_attributes: create_attribute().addClass('subheading'),
```

Comment:
Okay so from the looks of this and what you've said, since your subheadings are h2's, to better optimise your code so that you don't have to keep doing ```addClass('subheading')``` every-time  you use a heading level that's 2 you should just add this class in your heading component like you did for the page title

Reply:
done